package.json

```diff
@@ -22,41 +22,6 @@
   },
   "homepage": "https://github.com/HigherHelp/api#readme",
   "dependencies": {
-    "compression": "^1.7.4",
-    "cookie-parser": "^1.4.6",
-    "cors": "^2.8.5",
-    "express": "^4.19.2",
-    "helmet": "^7.1.0",
-    "lodash-es": "^4.17.21",
-    "morgan": "^1.10.0",
-    "winston": "^3.13.0"
-  },
-  "devDependencies": {
-    "eslint": "^8.33.0",
-    "eslint-config-airbnb-base": "15.0.0",
-    "eslint-config-prettier": "8.5.0",
-    "eslint-import-resolver-custom-alias": "^1.3.2",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-jest": "^27.2.2",
-    "eslint-plugin-no-floating-promise": "^1.0.2",
-    "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-prettier": "4.2.1",
-    "eslint-plugin-unicorn": "^45.0.2",
-    "lint-staged": "^15.2.5",
-    "nodemon": "^3.1.2",
-    "prettier": "2.8.0",
-    "simple-git-hooks": "^2.11.1"
-  },
-  "simple-git-hooks": {
-    "pre-commit": "npx lint-staged",
-    "pre-push": "yarn run format",
-    "post-merge": "yarn install"
-  },
-  "lint-staged": {
-    "*.{js,jsx}": "eslint --fix"
-  },
-  "volta": {
-    "node": "22.2.0",
-    "yarn": "1.22.22"
```

Comment:
Yeah I saw that there was an error because my branch was outdated, an it was showing that a merge is required to fix it so I just merged it manually.

Can I just simply run `yarn install` or do I need to manually put the code back in before doing so?